### PR TITLE
plausible.io analytics support in Docker image

### PR DESCRIPTION
**Pull request by dannymandel**
Docker side of https://github.com/isamplesorg/isamples_inabox/issues/118

### CODE REVIEW

**Comment by dannymandel** [comment]
@datadavev -- config for each box so it knows which metrics domain to report under

File: .env.geome [hunk 1]
```diff
@@ -5,3 +5,4 @@ SOLR_VOLUME_NAME=solrdata_geome
 CRONTAB_FILE=crontab.geome
 UVICORN_ROOT_PATH=geome
 ISB_HOST=henry.cyverse.org
+ENV_FILE=geome.isb_web_config.env
```
Comment (dannymandel) [inline .env.geome:8]: Have a new parameter to the docker build, which `.env` file to use when building the image.  Each `.env` flavor has a different metrics domain.

File: README.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ Then cd up to the isb docker directory and initialize git lfs:
 
 Then cd back to the root:
 `cd ..`
-and run the docker build for the flavor you are interested in: `docker-compose --env-file .env.opencontext up --build`
+and run the docker build for the flavor you are interested in: `docker-compose --env-file .env.opencontext -p isamples_docker_opencontext up --build`
```
Comment (dannymandel) [inline README.md:31]: This was just an earlier error I noticed when I tried to follow the documentation here when testing.

File: docker-compose.yml [hunk 3]
```diff
@@ -21,7 +21,9 @@ services:
         build:
             context: ./isb/
             args:
-                CRONTAB_FILE: ${CRONTAB_FILE}        
+                CRONTAB_FILE: ${CRONTAB_FILE}
+                ENV_FILE: ${ENV_FILE}
```
Comment (dannymandel) [inline docker-compose.yml:25]: New argument

File: isb/Dockerfile [hunk 4]
```diff
@@ -22,6 +22,11 @@ COPY ./cron/ .
 ARG CRONTAB_FILE=crontab.opencontext
 RUN echo "CRONTAB FILE IS $CRONTAB_FILE"
 COPY ./cron/$CRONTAB_FILE /etc/cron.d/update_crontab
+
+ARG ENV_FILE=opencontext.isb_web_config.env
+RUN echo "ENV FILE IS $ENV_FILE"
+COPY ./isb_web_config/$ENV_FILE ./isb_web_config.env
```
Comment (dannymandel) [inline isb/Dockerfile:28]: And this is where we copy the file argument to the container

**Review by datadavev — COMMENTED** [review]
This all looks great.

**Pull request merged**